modules/hybridBidAdapter.js

```diff
@@ -0,0 +1,198 @@
+import * as utils from '../src/utils'
+import { registerBidder } from '../src/adapters/bidderFactory'
+import { auctionManager } from '../src/auctionManager'
+import { BANNER, VIDEO } from '../src/mediaTypes'
+import {Renderer} from '../src/Renderer';
+import find from 'core-js/library/fn/array/find';
+
+const BIDDER_CODE = 'hybrid';
+const DSP_ENDPOINT = 'https://hbe198.hybrid.ai/prebidhb';
+const TRAFFIC_TYPE_WEB = 1;
+const PLACEMENT_TYPE_BANNER = 1;
+const PLACEMENT_TYPE_VIDEO = 2;
+const TTL = 60;
+const RENDERER_URL = '//acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
```

Comment:
If possible can we just use only `https`

It is a requirement for exchange endpoints to be https, and while this is just a renderer, I still think it would be a good idea to have it as https.

Reply:
We fixed it